packages/flutter/test/widgets/widget_inspector_test.dart

```diff
@@ -2827,14 +2827,37 @@ class _TestWidgetInspectorService extends TestWidgetInspectorService {
 
     test('devToolsInspectorUri test', () {
       activeDevToolsServerAddress = 'http://127.0.0.1:9100';
+      connectedVmServiceUri = 'http://127.0.0.1:55269/798ay5al_FM=/';
       expect(
-        WidgetInspectorService.instance.devToolsInspectorUri(
-          Uri.parse('http://127.0.0.1:55269/798ay5al_FM=/'),
-          'inspector-0',
-        ),
+        WidgetInspectorService.instance.devToolsInspectorUri('inspector-0'),
         equals('http://127.0.0.1:9100/#/inspector?uri=http%3A%2F%2F127.0.0.1%3A55269%2F798ay5al_FM%3D%2F&inspectorRef=inspector-0'),
       );
     });
+
```

Comment:
ignore this code. reviewed in https://github.com/flutter/flutter/pull/74455 - but this cl was built on top to avoid merge conflicts. Will rebase once https://github.com/flutter/flutter/pull/74455 lands.